components/constellation/constellation.rs

```diff
@@ -1783,7 +1803,8 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
         // Send to the explicitly focused pipeline (if it exists), or the root
         // browsing context's current pipeline. If neither exist, fall back to sending to
         // the compositor below.
-        let root_pipeline_id = self.browsing_contexts.get(&self.root_browsing_context_id)
+        let root_browsing_context_id = BrowsingContextId::from(self.root_browsing_context_id);
+        let root_pipeline_id = self.browsing_contexts.get(&root_browsing_context_id)
```

Comment:
I wonder if we could replace these occurrences with:
```
self.browsing_contexts.get(&self.root_browsing_context_id.into())
```
since `From<T> for U` implies `Into<U> for T`